internal/cli.go

```diff
@@ -0,0 +1,15 @@
+package internal
+
+import (
+	"io"
+
+	"github.com/influxdata/influx-cli/v2/internal/api"
+)
+
+type CLI struct {
```

Comment:
Decided I didn't like the custom `context` wrapper, made this as the replacement.

Self-review: needs a doc comment